environment.py

```diff
@@ -64,14 +64,19 @@ def env() -> Mapping[str, Optional[str]]:
         # TDR source = 'tdr:', Google Cloud project name,
         #              ':', ( 'dataset' | 'snapshot' ),
         #              '/', TDR dataset or snapshot name,
-        #              ':', [ UUID prefix [ '/', Partition prefix length ] ] ;
+        #              ':', [ UUID prefix ], '/', partition prefix length ;
```

Comment:
AFAICT, the partition prefix length is still optional.

Reply:
Got these changes mixed up from the EBNF of #3530, fixed the capitalization.